test/relay.transaction.test.ts

```diff
@@ -0,0 +1,58 @@
+import { RelayingServices, SmartWallet } from '../src';
+import { MockRelayingServices, Web3Mock } from './mock';
+import { TransactionConfig, TransactionReceipt } from 'web3-core';
+import Expect = jest.Expect;
+import {
+    MOCK_CONTRACT_ADDRESS,
+    MOCK_ADDRESS,
+    MOCK_SMART_WALLET_ADDRESS,
+    MOCK_TRANSACTION_RECEIPT
+} from './constants';
+
+declare const expect: Expect;
+const transaction: TransactionConfig = {
+    from: MOCK_ADDRESS,
+    to: MOCK_CONTRACT_ADDRESS,
+    value: 1
+};
+const smartWallet: SmartWallet = {
+    address: MOCK_SMART_WALLET_ADDRESS,
+    index: 0,
+    deployed: true
+};
+describe('Tests for relay transaction', () => {
+    let sdk: RelayingServices;
+
+    beforeEach(async () => {
+        sdk = new MockRelayingServices();
+        sdk.initialize({});
```

Comment:
I think we should await here
```suggestion
        await sdk.initialize({});
```

Reply:
Fixed

Reply:
It looks like we removed the `sdk.initialize()` call. Isn't it necessary?

Reply:
It was remove by mistake, Fixed.